doc/user-manual.md

```diff
@@ -1,216 +1,457 @@
+
+### Tracing Component
+Component Type | Component | Reference | 
+--- | ---| --- |
+HTTP Client | `RestTemplate`、 `WebClient`、 `FeignClient` | [brave-instrumentation-http](https://github.com/openzipkin/brave/tree/master/instrumentation/http)
+HTTP Server | `Servlet`、`Filter` | [brave-instrumentation-http](https://github.com/openzipkin/brave/tree/master/instrumentation/http)
+DataBase | `JDBC` | [Brave](https://github.com/openzipkin/brave/tree/master/brave)
+Cache | `Jedis`、`Lettuce` | [Brave](https://github.com/openzipkin/brave/tree/master/brave)
+Message | `RabbitMQ`、`Kafka` | [brave-instrumentation-messaging](https://github.com/openzipkin/brave/tree/master/instrumentation/messaging) 、[Brave Kafka instrumentation](https://github.com/openzipkin/brave/tree/master/instrumentation/kafka-clients)
+Logging | `Log4j2`、`Logback` | [brave-context-log4j2](https://github.com/openzipkin/brave/tree/master/context/log4j2) 、[brave-context-slf4j](https://github.com/openzipkin/brave/tree/master/context/slf4j)
+
+### Custom Span Tag
+
+#### JDBC
+Tag | Description |
+--- | ---|
+sql | Sql text in user application |
+local-component | Default value = 'database' |
+url | Connection information. Example: `jdbc:mysql://localhost:3306/db_demo` |
+error | SQLException information |
+
+#### Cache
+Tag | Description |
+--- | ---|
+redis.method | Redis command. Example: `MGET`、`GET` |
+
+#### RabbitMQ Producer And Consumer
+Tag | Description |
+--- | ---|
+rabbit.exchange | RabbitMQ exchange |
+rabbit.routing_key | RabbitMQ routingKey |
+rabbit.queue | RabbitMQ routingKey |
+
+#### Kafka Producer And Consumer
+Tag | Description |
+--- | ---|
+kafka.key | Kafka consumer record Key |
+kafka.topic | Kafka topic |
+kafka.broker | Kafka url |
+
+## Metric
+EaseAgent use [io.dropwizard.metrics](https://github.com/dropwizard/metrics) to collect metric information.
+
+
+### Metric Field
+EaseAgent output metric data to kafka. The data stored in kafka is in JSON format.
+
+For Example: EaseAgent collect metric of HTTP Request. The collected metric data are as follows:
+```json
+{
+  "m15err" : 0,
+  "m5err" : 0,
+  "cnt" : 1,
+  "url" : "GET \/",
+  "m5" : 0.050990000000000001,
+  "max" : 823,
+  "mean" : 823,
+  "p98" : 823,
+  "errcnt" : 0,
+  "host_name" : "akwei",
+  "min" : 823,
+  "category" : "application",
+  "system" : "none",
+  "type" : "http-request",
+  "mean_rate" : 0,
+  "p99" : 823,
+  "p95" : 823,
+  "m15" : 0.12681999999999999,
+  "timestamp" : 1621567320892,
+  "service" : "unknown-service",
+  "m1" : 0.00022000000000000001,
+  "m5errpct" : 0,
+  "p25" : 823,
+  "p75" : 823,
+  "p50" : 823,
+  "host_ipv4" : "192.168.2.5",
+  "m1errpct" : 0,
+  "m15errpct" : 0,
+  "m1err" : 0,
+  "p999" : 823
+}
+```
+
+For different components, the fields contained in JSON are as follows:
+
+#### HTTP Request
```

Comment:
HTTP Request schema describes key metrics of service APIs, which include:
- Throughput (M1, M5, M15),
- Error throughput (ERROR M1, M5, M15)
- Latency (p99, p98,p95, p75,p50...)

Reply:
I fixed, and remove some unused fields